CHANGELOG.md

```diff
@@ -31,6 +31,22 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 -
 
+## [0.1.50] - 2022-04-05
+
+### Enhancements
+
+- 
+
+### Fixes
+
+- Fix issues of unpausing machine config pool too soon after apply remediations,
+  added a check to check if kubelet has been fully rendered into machine config
+  before unpause affected pool. [1]
```

Comment:
nit: I'd try and include the outcome here if possible (helps justify why we made the change).

For example:

The compliance-operator now waits until the kubelet configuration is fully rendered before unpausing the pool to avoid kubelets getting stuck in a `NotReady` state.